src/index.html

```diff
@@ -7,8 +7,58 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header_container">
+      <a href="#home" class="logo_link"><img src="images/logo.png" alt="logo icon"></a>
+      <nav>
+        <ul class="nav_items_list">
+          <li class="nav_items">
+            <a href="#apple" class="nav__link is-active">
+              Apple
+            </a>
+          </li>
+          <li class="nav_items">
```

Comment:
```suggestion
      <a href="#home" class="logo_link">
        <img src="images/logo.png" alt="logo icon">
      </a>
      
      <nav>
        <ul class="nav_items_list">
          <li class="nav_items">
            <a href="#apple" class="nav__link is-active">
              Apple
            </a>
          </li>
          
          <li class="nav_items">
```
fix code style all over the page